app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,8 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = @user.posts.order(created_at: :desc)
+    # @posts = @user.posts.order(created_at: :desc)
```

Comment:
- [ ] [OPTIONAL] You can remove this previous code as it is unnecessary now.

Reply:
Thanks, bro. I removed it from the branch.